iree/test/e2e/xla_ops/BUILD

```diff
@@ -44,7 +44,6 @@ iree_check_single_backend_test_suite(
             "dynamic_update_slice.mlir",
             "exponential.mlir",
             "exponential_minus_one.mlir",
-            "fft.mlir",
```

Comment:
Does it not work on CUDA? I think after PR #6615 it should work.

Reply:
I forgot to add it to `exclude` list and attached the bug ID. We will have an index buffer for doing data reordering. However, it looks like CUDA side does not support index buffer (tensor<1000xindex>) lowering. See https://github.com/google/iree/issues/6604